[release-1.4] update RHBK config docs with security consideration (#869)
redhat-developer/red-hat-developers-documentation-rhdh: modules/authentication/proc-enabling-authentication-with-rhbk.adoc

--

.Security consideration

> If multiple valid refresh tokens are issued due to frequent refresh token requests, older tokens will remain valid until they expire. To enhance security and prevent potential misuse of older tokens, we recommend enabling a refresh token rotation strategy in your {rhbk} realm.

Reviewer: ```suggestion
If multiple valid refresh tokens are issued due to frequent refresh token requests, older tokens will remain valid until they expire. To enhance security and prevent potential misuse of older tokens, enable a refresh token rotation strategy in your {rhbk} realm.
```
Avoid recommendations (i.e. "we recommend" / "is recommended") and simply state what the user needs to do and why.

> . Navigate to the **Configure** > **Realm Settings**.
> . Enable the **Revoke Refresh Token** option.

Reviewer: ```suggestion
. Click **Configure** > **Realm Settings** and enable the **Revoke Refresh Token** option.
```

First, orient the user: _From the [page, menu, etc.], click **Configure** > **Realm Settings**_

How does the user enable the  **Revoke Refresh Token** option? Do they click a toggle? Or select from a drop-down menu? Let's explain that.

--

.Security consideration
If multiple valid refresh tokens are issued due to frequent refresh token requests, older tokens will remain valid until they expire. To enhance security and prevent potential misuse of older tokens, enable a refresh token rotation strategy in your {rhbk} realm.

> . From the navigation menu on the left-hand side, click **Configure** > **Realm Settings**.

Reviewer: ```suggestion
. From the navigation menu, click **Configure** > **Realm Settings**.
```
remove "pointing" language

> . Navigate to the **Tokens** tab.
> . Toggle on the **Revoke Refresh Token** option.

Reviewer: ```suggestion
. Navigate to the **Tokens** tab and toggle on the **Revoke Refresh Token** option.
```
combine these steps
